compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.td

```diff
@@ -67,6 +67,27 @@ def FLOW_DispatchWorkgroupsOp : FLOW_PureOp<"dispatch.workgroups", [
     computes the actual workgroup count based on target information. The
     workload is not limited to the 3D XYZ grid dispatch of the workgroup count
     and can contain any number of parameters used to compute it.
+
+    ```mlir
+    %r = flow.dispatch.workgroups[%c5, %c5](%0, %1)
+        : (tensor<5x5xf32>, tensor<5xf32>) -> tensor<5x5xf32> =
+              (%arg0: !flow.dispatch.tensor<readonly:5x5xf32>,
+               %arg1: !flow.dispatch.tensor<readonly:5xf32>,
+               %arg2: !flow.dispatch.tensor<writeonly:5x5xf32>) {
+      ...
+    }
+    ```
+
+    The number of results of the operation is equal to the number of results
+    in the type signature (`(tensor<5x5xf32>, tensor<5xf32>) -> tensor<5x5xf32>`).
+    Each tensor argument and result in the type signature has a corresponding
+    block argument of type `!flow.dispatch.tensor`. Furthermore, each argument
+    has a corresponding `arguments` operand.
+
+    There are no `arguments` operands for results, but a result can be tied an
+    argument by writing the argument operand's SSA value instead of its type:
+    E.g., in the above example, `-> %0` ties the first argument to the result.
```

Comment:
I don't see `-> %0` in the example above. Is it missing?

Reply:
Updated the comment.